<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tmh_main_window</class>
 <widget class="QMainWindow" name="tmh_main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>지수 산정 도구 V1.0</string>
  </property>
  <widget class="QWidget" name="tmh_central_box">
   <widget class="QGroupBox" name="tmh_standard_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>321</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>기준 종</string>
    </property>
    <widget class="QTableWidget" name="tmh_standard_box_result">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>199</y>
       <width>300</width>
       <height>201</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>이름</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>코드</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>비고</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="tmh_standard_box_Input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tmh_standard_box_title1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>종별 코드 파일</string>
     </property>
    </widget>
    <widget class="QLabel" name="tmh_standard_box_title2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>종별 코드 요약</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tmh_standard_box_outline">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>300</width>
       <height>71</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>57</number>
     </attribute>
     <column>
      <property name="text">
       <string>전체</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>4</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>5</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>미할당</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="tmh_standard_box_title3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>종별 코드표</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tmh_menu_options_type_code">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>종별 코드 선택</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tmh_validate_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>9</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>분석 대상</string>
    </property>
    <widget class="QLabel" name="tmh_validate_box_title1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>데이터 파일</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tmh_validate_box_validate_btn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>검증</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tmh_validate_box_select_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>선택</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tmh_validate_box_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>파일을 선택하세요.</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tmh_validate_box_result">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>101</y>
       <width>301</width>
       <height>161</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>정점</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>결과</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="tmh_validate_box_title2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>데이터 검증 결과표</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tmh_select_index_box">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>661</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>지수 산정</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>29</y>
       <width>641</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="tmh_index_tab">
      <item row="0" column="0">
       <widget class="QPushButton" name="tmh_index_AMBI">
        <property name="text">
         <string>AMBI</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="tmh_index_ISEP">
        <property name="text">
         <string>ISEP Index</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="tmh_index_SW">
        <property name="text">
         <string>Shannon-Wiener Index</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tmh_result_box">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>661</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>지수 산정 결과</string>
    </property>
    <widget class="QTabWidget" name="tmh_result_tab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>641</width>
       <height>571</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tmh_AZTI_tab">
      <attribute name="title">
       <string>AZTI's Marine Biotic Index</string>
      </attribute>
      <widget class="QTableWidget" name="tmh_AZTI_result">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>800</width>
         <height>511</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>250</number>
       </attribute>
       <column>
        <property name="text">
         <string>정점</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>지수</string>
        </property>
       </column>
      </widget>
      <widget class="QProgressBar" name="tmh_AZTI_bar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>520</y>
         <width>611</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tmh_inverse_SW_tab">
      <attribute name="title">
       <string>Inverse function of Shannon-Wiener Evenness Proportion</string>
      </attribute>
      <widget class="QTableWidget" name="tmh_inverse_SW_result">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>611</width>
         <height>511</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>250</number>
       </attribute>
       <column>
        <property name="text">
         <string>정점</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>지수</string>
        </property>
       </column>
      </widget>
      <widget class="QProgressBar" name="tmh_inverse_SW_bar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>520</y>
         <width>611</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tmh_SW_tab">
      <attribute name="title">
       <string>Shannon-Wiener Index</string>
      </attribute>
      <widget class="QTableWidget" name="tmh_SW_result">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>611</width>
         <height>511</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>170</number>
       </attribute>
       <column>
        <property name="text">
         <string>정점</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>지수(ln)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>지수(log)</string>
        </property>
       </column>
      </widget>
      <widget class="QProgressBar" name="tmh_SW_bar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>520</y>
         <width>611</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>그래프 보기</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="tmh_status_bar"/>
  <widget class="QMenuBar" name="tmh_menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="tmh_menu_file">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="tmh_menu_file_new"/>
    <addaction name="tmh_menu_file_open"/>
    <addaction name="tmh_menu_file_recent_form"/>
    <addaction name="separator"/>
    <addaction name="tmh_menu_file_save"/>
    <addaction name="tmh_menu_file_save_as"/>
    <addaction name="tmh_menu_file_save_all"/>
    <addaction name="separator"/>
    <addaction name="tmh_menu_file_close"/>
   </widget>
   <widget class="QMenu" name="tmh_menu_options">
    <property name="title">
     <string>선택</string>
    </property>
    <addaction name="tmh_menu_options_workdir"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="tmh_menu_help">
    <property name="title">
     <string>도움말</string>
    </property>
    <addaction name="tmh_menu_help_help"/>
    <addaction name="separator"/>
    <addaction name="tmh_menu_help_about"/>
   </widget>
   <addaction name="tmh_menu_file"/>
   <addaction name="tmh_menu_options"/>
   <addaction name="tmh_menu_help"/>
  </widget>
  <action name="tmh_menu_help_about">
   <property name="text">
    <string>이 도구에 관하여</string>
   </property>
  </action>
  <action name="tmh_menu_options_workdir">
   <property name="text">
    <string>작업 디렉토리 선택</string>
   </property>
  </action>
  <action name="tmh_menu_file_new">
   <property name="text">
    <string>신규</string>
   </property>
  </action>
  <action name="tmh_menu_file_open">
   <property name="text">
    <string>열기</string>
   </property>
  </action>
  <action name="tmh_menu_file_recent_form">
   <property name="text">
    <string>최근 작업</string>
   </property>
  </action>
  <action name="tmh_menu_file_save">
   <property name="text">
    <string>저장</string>
   </property>
  </action>
  <action name="tmh_menu_file_save_as">
   <property name="text">
    <string>다른 이름으로 저장</string>
   </property>
  </action>
  <action name="tmh_menu_file_save_all">
   <property name="text">
    <string>전체 저장</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>종별 코드 선택</string>
   </property>
  </action>
  <action name="tmh_menu_help_help">
   <property name="text">
    <string>도움말</string>
   </property>
  </action>
  <action name="tmh_menu_file_close">
   <property name="text">
    <string>닫기</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tmh_menu_file_new</sender>
   <signal>triggered()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_new_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_menu_file_open</sender>
   <signal>triggered()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_open_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_menu_file_recent_form</sender>
   <signal>triggered()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_recent_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_menu_file_save</sender>
   <signal>triggered()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_save_this()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_menu_file_save_as</sender>
   <signal>triggered()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_custom_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_menu_file_save_all</sender>
   <signal>triggered()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_save_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_menu_file_close</sender>
   <signal>triggered()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_menu_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_menu_options_workdir</sender>
   <signal>triggered()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_set_workdir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action</sender>
   <signal>triggered()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_set_type_code()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_menu_help_help</sender>
   <signal>triggered()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_open_help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_menu_help_about</sender>
   <signal>triggered()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_open_info()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_validate_box_select_btn</sender>
   <signal>clicked()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_open_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_index_SW</sender>
   <signal>clicked()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_calc_sw_index()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_index_AMBI</sender>
   <signal>clicked()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_calc_ambi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_index_ISEP</sender>
   <signal>clicked()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_calc_isep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_validate_box_validate_btn</sender>
   <signal>clicked()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_check_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tmh_menu_options_type_code</sender>
   <signal>clicked()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_set_type_code()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>tmh_main_window</receiver>
   <slot>tmh_show_graph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>1017</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tmh_new_file()</slot>
  <slot>tmh_open_file()</slot>
  <slot>tmh_recent_file()</slot>
  <slot>tmh_save_this()</slot>
  <slot>tmh_custom_save()</slot>
  <slot>tmh_save_all()</slot>
  <slot>tmh_set_workdir()</slot>
  <slot>tmh_set_type_code()</slot>
  <slot>tmh_open_help()</slot>
  <slot>tmh_open_info()</slot>
  <slot>tmh_menu_close()</slot>
  <slot>tmh_check_data()</slot>
  <slot>tmh_change_type_code()</slot>
  <slot>tmh_calc_ambi()</slot>
  <slot>tmh_calc_isep()</slot>
  <slot>tmh_calc_sw_index()</slot>
  <slot>tmh_show_graph()</slot>
 </slots>
</ui>
